app/frontend/src/views/user/SubmissionDraftEdit.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <UserSubmission :saved="sv" :submissionId="s" :readOnly="false" />
+  </div>
```

Comment:
Don't need the wrapper `<div>` elements.

Reply:
Fixed